### DISCUSSION TITLE
Schema渲染事件问题

### DISCUSSION THREAD

**Discussion by stwflyfox** [opentiny/tiny-engine#902] [Q&A]
打算自己写一个用Schema渲染页面，参考tinyEngine里的render.js，不知道如何把组件中定义的事件加到render里，是在 h() 函数中吗？ 还有页面的onMounted事件要加到哪里？

**Comment by chilingling** [comment]
template 模板对应的是 vue 的 [render 函数](https://vuejs.org/api/render-function.html#render-function-apis)。
onMounted 建议放到 RenderMain 函数中集中处理。也就是 render.js 的上层 `RenderMain.js ` 文件中的 `setup` 函数，`onMounted` 回调中解析页面的 `onMounted` 逻辑然后执行。

**Comment by stwflyfox** [comment]
thanks
